<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElectrocardiogramGUIClass</class>
 <widget class="QMainWindow" name="ElectrocardiogramGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElectrocardiogramGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="btnVLayout">
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Load ECG file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSmooth">
        <property name="text">
         <string>Smooth Signal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save Signal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QChartView" name="chart"/>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="infoGridLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblCount">
        <property name="text">
         <string>Count</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtCount">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSum">
        <property name="text">
         <string>Sum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtSum">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadFile"/>
    <addaction name="actionUnloadFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoadFile">
   <property name="text">
    <string>Load ECG File</string>
   </property>
  </action>
  <action name="actionUnloadFile">
   <property name="text">
    <string>Unload ECG File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ElectrocardiogramGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnActionQuitTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoadFile</sender>
   <signal>triggered()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnLoadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUnloadFile</sender>
   <signal>triggered()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoad</sender>
   <signal>clicked()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnLoadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnSaveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSmooth</sender>
   <signal>clicked()</signal>
   <receiver>ElectrocardiogramGUIClass</receiver>
   <slot>OnSmoothSignal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
